script/Arcueid/Day 6/06_00_ARC06_3_1.txt

```diff
@@ -225,7 +226,7 @@ That girl was nothing but trouble, but even so, it was... well, kinda fun to spe
 [sha:da7e29a27b4dbf7077cae9615fd32b77877855b8]{
 -- Page 276, Offset 14580.
 -- 一度殺されかけたっていうのに、何が未練なんだか。
-I nearly got killed, why on earth am I still pining after her?
+I nearly got killed, why on Earth am I still pining after her?
```

Comment:
Earth only capitalized when talking about it as a celestial body 
https://www.dictionary.com/e/earth/

Reply:
Is that not what it's referring to here? I'm aware it's just a phrase and people aren't literally asking about planets or dirt, but this is in the same vein as "What in the world" and such.

Reply:
Nope, it's not a proper noun in expressions like down to earth, what on earth, etc.